lib/system_description_store.rb

```diff
@@ -114,6 +114,26 @@ def rename(from, to)
     FileUtils.mv(description_path(from), description_path(to))
   end
 
+  def swap(description_name_1, description_name_2)
+    if !list.include?(description_name_1)
+      raise Machinery::Errors::SystemDescriptionNotFound.new(
+        "System description \"#{description_name_1}\" does not exist."
+      )
+    end
+
+    if !list.include?(description_name_2)
+      raise Machinery::Errors::SystemDescriptionNotFound.new(
+        "System description \"#{description_name_2}\" does not exist."
+      )
+    end
```

Comment:
The test if a description exists is repeated twice in this method and more times in other methods. We should extract that in an own method. But we can merge this pull request first.